File: app/src/main/java/tech/leonam/resultadodocertame/model/interfaces/InterfaceCadastraTurma.java
package tech.leonam.resultadodocertame.model.interfaces;

import android.content.Context;

import tech.leonam.resultadodocertame.model.entidade.TurmaEntidade;

public interface InterfaceCadastraTurma {

    boolean cadastrar(TurmaEntidade turmaEntidade, Context context);
}

File: app/src/main/java/tech/leonam/resultadodocertame/model/interfaces/InterfaceCadastraTurma.kt
package tech.leonam.resultadodocertame.model.interfaces

import android.content.Context
import tech.leonam.resultadodocertame.modelView.entidade.TurmaEntidade

interface InterfaceCadastraTurma {
    fun cadastrar(turmaEntidade: TurmaEntidade?, context: Context?): Boolean
}